doc/src/Howto_walls.rst

```diff
@@ -67,5 +67,5 @@ rotate.
 
 The only frictional idealized walls currently in LAMMPS are flat or
 curved surfaces specified by the :doc:`fix wall/gran <fix_wall_gran>`
-command.  At some point we plan to allow regoin surfaces to be used as
+command.  At some point we plan to allow region surfaces to be used as
```

Comment:
nice catch! you should also delete the wrong word from ` doc/utils/sphinx-config/false_positives.txt`

Reply:
Oh, I see, good point, done, thanks.